tests/schemas/api_v3/transaction.schema.js

```diff
@@ -18,9 +18,9 @@ import Joi from 'joi';
 const regex = require('./regex');
 
 const sender = {
-	address: Joi.string().pattern(regex.ADDRESS_BASE32).required(),
-	publicKey: Joi.string().pattern(regex.PUBLIC_KEY).required(),
-	name: Joi.string().pattern(regex.NAME).required(),
+	address: Joi.string().pattern(regex.ADDRESS_LISK32).required(),
+	publicKey: Joi.string().pattern(regex.PUBLIC_KEY).optional(),
```

Comment:
I think `publicKey` for the transaction sender is always available. Please double check.